SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -389,7 +396,14 @@ - (void)didEnterStateVideoStreamStarting {
 
         // Apply customEncoderSettings here. Note that value from HMI (such as maxBitrate) will be overwritten by custom settings.
         for (id key in self.customEncoderSettings.keyEnumerator) {
-            self.videoEncoderSettings[key] = [self.customEncoderSettings valueForKey:key];
+            if ([(NSString *)key isEqualToString:(__bridge NSString *)kVTCompressionPropertyKey_ExpectedFrameRate] == YES) {
```

Comment:
One should not compare a bool value with YES or NO, especially with YES:
1. The method -isEqualToString: already produces a bool value (YES or NO) which can be used directly without comparing it one more time to YES.
2. Let's take a more sophisticated example:
const BOOL bValue = (BOOL)2; // true condition
if (bValue) { /* true condition works */ }
if (bValue == YES) { /* true condition does not work */ }

Reply:
Thanks for the review. Fixed as suggested.